src/CHANGES.md

```diff
@@ -1,6 +1,6 @@
 Changelog
 
-## [Unreleased](https://github.com/bids-standard/bids-specification/tree/HEAD)
+## [1.1.2](https://github.com/bids-standard/bids-specification/tree/1.1.2)
```

Comment:
```suggestion
## [Unreleased](https://github.com/bids-standard/bids-specification/tree/HEAD)

## [1.1.2](https://github.com/bids-standard/bids-specification/tree/1.1.2)
```

Reply:
@franklin-feingold would this change be preserved by the automatic changelog scripts?

Reply:
Actually, I don't think this will work. @choldgraf the CI scripts will add the "unreleased" section after this is merged.

Reply:
See the protocol: https://github.com/bids-standard/bids-specification/blob/master/Release_Protocol.md

Reply:
sounds good, +1 to merge from me then

Reply:
This change would not be preserved by the automatic changelog scripts. There will be an unreleased section as soon as another PR is merged after the release (this PR will not trigger an unreleased section generation, the next one will). 

The release appears to have worked! My changelog generator scripts didn't run.